tools/amd_build/build_amd.py

```diff
@@ -142,10 +136,18 @@
                     f.flush()
                     os.fsync(f)
 
+# Check if the compiler is hip-clang.
+def is_hip_clang():
+    try:
+        hip_path = os.getenv('HIP_PATH', '/opt/rocm/hip')
+        return 'HIP_COMPILER=clang' in open(hip_path + '/lib/.hipInfo').read()
+    except Exception:
```

Comment:
What exceptions are you expecting? Is it IOError?

Reply:
Yes, that'd be the standard case - do you want me to specialize?

Reply:
yes please